packages/dds/tree/src/class-tree/schemaFactory.ts

```diff
@@ -286,7 +286,7 @@ export class SchemaFactory<TScope extends string = string, TName extends number
 	 * Define a structurally typed {@link TreeNodeSchema} for a {@link TreeMapNodeBase}.
 	 *
 	 * @remarks
-	 * The {@link TreeNodeSchemaIdentifier} for this Map is defined as a function of the provided types.
+	 * The unique identifier for this Map is defined as a function of the provided types.
```

Comment:
There were ~5 doc comments that contained {@link} tags to non-public types that needed to be fixed.